### ISSUE TITLE
Windows MathLoadTest_autosimd_5m Illegal instruction

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9#19377]
https://openj9-jenkins.osuosl.org/job/Test_openjdk8_j9_sanity.system_x86-32_windows_Nightly_testList_0/757 - [win2012x64-openj9-1a](https://openj9-jenkins.osuosl.org/computer/win2012x64-openj9-1a)
MathLoadTest_autosimd_5m_2
`variation: Mode1000` - no options 

https://openj9-artifactory.osuosl.org/artifactory/ci-openj9/Test/Test_openjdk8_j9_sanity.system_x86-32_windows_Nightly_testList_0/757/system_test_output.tar.gz
```
02:04:48  MLT stderr Type=Illegal instruction vmState=0x00000000
02:04:48  MLT stderr J9Generic_Signal_Number=00000010 ExceptionCode=c000001d ExceptionAddress=282A0E27 ContextFlags=0001007f
02:04:48  MLT stderr Handler1=740A53B0 Handler2=73FC54D0
02:04:48  MLT stderr EDI=00000001 ESI=00000000 EAX=25702000 EBX=00000008
02:04:48  MLT stderr ECX=20C10F10 EDX=20C10F58
02:04:48  MLT stderr EIP=282A0E27 ESP=257516D0 EBP=25684200 EFLAGS=00010297
02:04:48  MLT stderr GS=002B FS=0053 ES=002B DS=002B
02:04:48  MLT stderr Module=
02:04:48  MLT stderr Module_base_address=28280000 Offset_in_DLL=00020e27
02:04:48  MLT stderr 
02:04:48  MLT stderr Compiled_method=net/adoptopenjdk/test/autosimd/AutoSIMDTestLong.testSimpleBinary(Lnet/adoptopenjdk/test/autosimd/BinaryOpSIMDLong;)V
02:04:48  MLT stderr Target=2_90_20240425_818 (Windows Server 2012 R2 6.3 build 9600)
02:04:48  MLT stderr CPU=x86 (4 logical CPUs) (0x3fff77000 RAM)
02:04:48  MLT stderr ----------- Stack Backtrace -----------
02:04:48  MLT stderr Unhandled exception
02:04:48  MLT stderr Type=Illegal instruction vmState=0x00000000
02:04:48  MLT stderr J9Generic_Signal_Number=00000010 ExceptionCode=c000001d ExceptionAddress=282A0E27 ContextFlags=0001007f
02:04:48  MLT stderr Handler1=740A53B0 Handler2=73FC54D0
02:04:48  MLT stderr EDI=00000001 ESI=00000000 EAX=25702000 EBX=00000008
02:04:48  MLT stderr ECX=20CC1688 EDX=20CC16D0
02:04:48  MLT stderr EIP=282A0E27 ESP=2575CFA8 EBP=22184600 EFLAGS=00010297
02:04:48  MLT stderr GS=002B FS=0053 ES=002B DS=002B
02:04:48  MLT stderr Module=
02:04:48  MLT stderr Module_base_address=28280000 Offset_in_DLL=00020e27
02:04:48  MLT stderr 
02:04:48  MLT stderr Compiled_method=net/adoptopenjdk/test/autosimd/AutoSIMDTestLong.testSimpleBinary(Lnet/adoptopenjdk/test/autosimd/BinaryOpSIMDLong;)V
02:04:48  MLT stderr Target=2_90_20240425_818 (Windows Server 2012 R2 6.3 build 9600)
02:04:48  MLT stderr CPU=x86 (4 logical CPUs) (0x3fff77000 RAM)
02:04:48  MLT stderr ----------- Stack Backtrace -----------
02:04:48  MLT stderr Unhandled exception
02:04:48  MLT stderr Type=Illegal instruction vmState=0x00000000
02:04:48  MLT stderr J9Generic_Signal_Number=00000010 ExceptionCode=c000001d ExceptionAddress=282A0E27 ContextFlags=0001007f
02:04:48  MLT stderr Handler1=740A53B0 Handler2=73FC54D0
02:04:48  MLT stderr EDI=00000001 ESI=00000000 EAX=25702000 EBX=00000008
02:04:48  MLT stderr ECX=20DDC600 EDX=20DDC648
02:04:48  MLT stderr EIP=282A0E27 ESP=25757340 EBP=22088000 EFLAGS=00010297
02:04:48  MLT stderr GS=002B FS=0053 ES=002B DS=002B
02:04:48  MLT stderr Module=
02:04:48  MLT stderr Module_base_address=28280000 Offset_in_DLL=00020e27
02:04:48  MLT stderr 
02:04:48  MLT stderr Compiled_method=net/adoptopenjdk/test/autosimd/AutoSIMDTestLong.testSimpleBinary(Lnet/adoptopenjdk/test/autosimd/BinaryOpSIMDLong;)V
02:04:48  MLT stderr Target=2_90_20240425_818 (Windows Server 2012 R2 6.3 build 9600)
02:04:48  MLT stderr CPU=x86 (4 logical CPUs) (0x3fff77000 RAM)
02:04:48  MLT stderr ----------- Stack Backtrace -----------
02:04:48  MLT stderr (0x282A0E27)
02:04:48  MLT stderr omrthread_get_category+0x1975 (0x74004AE5 [j9thr29+0x4ae5])
02:04:48  MLT stderr SwitchToThread+0x6 (0x76A44E86 [KERNELBASE+0x24e86])
02:04:48  MLT stderr SwitchToThread+0x6 (0x76A44E86 [KERNELBASE+0x24e86])
02:04:48  MLT stderr J9_GetInterface+0x17c6a (0x740C7D7A [j9vm29+0x97d7a])
02:04:48  MLT stderr J9_GetInterface+0x179ec (0x740C7AFC [j9vm29+0x97afc])
02:04:48  MLT stderr ---------------------------------------
02:04:48  MLT stderr (0x282A0E27)
02:04:48  MLT stderr (0x21FC5600)
02:04:48  MLT stderr (0x87780000)
02:04:48  MLT stderr ---------------------------------------
02:04:48  MLT stderr (0x282A0E27)
02:04:48  MLT stderr (0x21FC5600)
02:04:48  MLT stderr (0x87780000)
02:04:48  MLT stderr ---------------------------------------
```

Changes from the previous build
https://github.com/eclipse-openj9/openj9/compare/4998aab3c...70358c1a9fa
https://github.com/eclipse-openj9/openj9-omr/compare/723d2e4b63...e2d8e0fae97
https://github.com/ibmruntimes/openj9-openjdk-jdk8/compare/9fd4871da...6bc6c60e206

**Comment by hzongaro** [comment]
@BradleyWood, may I ask you to take a look at this?  Looking at this in windbg, the illegal instruction is this:

```
282a0e27 62f17e086f8cf208000000 vmovdqu32 xmm1, xmmword ptr [edx+esi*8+8
```

**Comment by BradleyWood** [comment]
The encoding of that instruction is valid, but it is not supported on 32-bit. Not sure how it got generated.

**Comment by BradleyWood** [comment]
I suppose we have an AVX-512 capable machine running a WIN-32 build which is why it was generated.

**Comment by pshipton** [comment]
Is this a recent regression?

**Comment by BradleyWood** [comment]
> Is this a recent regression?

I think it was probably introduced around a year and a half ago

**Comment by pshipton** [comment]
Seems it doesn't need to be fixed in 0.44 then.

**Comment by pshipton** [comment]
I'll close this as a dup of https://github.com/eclipse-openj9/openj9/issues/19408 even though this was opened first.
